docs/install-gitlab.md

```diff
@@ -22,7 +22,8 @@ frogbot-scan:
       # Creating fix pull requests will be triggered by any push to the default branch.
       # You can change it to any other branch you want, for example:
```

Comment:
You can change it to any other branch you want, for example:
-->
If you;d like a different branch to be scanned, replace $CI_DEFAULT_BRANCH in the line below with the name of branch, wrapped with quotes (").